src/stan_inference.jl

```diff
@@ -62,6 +62,7 @@ function stan_inference(prob::DiffEqBase.DEProblem,
                         # stan_sample keyword arguments
                         num_samples = 1000, num_warmups = 1000,
                         num_cpp_chains = 1, num_chains = 1, num_threads = 1,
+                        delta = 0.8,
```

Comment:
On a second thought, I wonder if it would be easier and possibly better to group all `stan_sample` keyword arguments in a `stan_kwargs`, similar to `mcmc_kwargs` (https://github.com/SciML/DiffEqBayes.jl/blob/a1c6b90ba0f4dfedc4e41fc56212709b254ecf19/src/dynamichmc_inference.jl#L103), and then just forward it to `stan_sample`? Or even just forward all `kwargs...`?

Then it would be easier to customize also other Stan settings.

Reply:
yeah that would probably be best